taskw/fields/depends.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+from distutils.version import LooseVersion
+
+import re
+import uuid
+
+from .base import DirtyableList, Field
+
+
+class DependsField(Field):
```

Comment:
Renamed this field since the name was potentially misleading, and given that we long ago made this field specific to the idiosyncrasies of 'depends', it's probably reasonable to name it to indicate that.